charts/fluent-bit/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: "v1"
 name: fluent-bit
-version: 2.3.2
+version: 2.3.3
```

New feature should bump to minor version.
```suggestion
version: 2.4.0
```